themes/socialbase/templates/node/field--node--body.html.twig

```diff
@@ -6,10 +6,27 @@
 
 {% else %}
 
-  <div class="body-text clearfix">
-    {% for item in items %}
-      {{ item.content }}
-    {% endfor %}
-  </div>
+  {%
+    set title_classes = [
+    'field--label',
+    label_display == 'visually_hidden' ? 'sr-only',
+  ]
+  %}
+
+  {% if label_hidden %}
+    <div class="body-text clearfix">
+      {% for item in items %}
+        {{ item.content }}
+      {% endfor %}
+    </div>
+  {% else %}
+    <div{{ title_attributes.addClass(title_classes) }}>{{ label }}</div>
+
+    <div class="body-text clearfix">
+      {% for item in items %}
+        {{ item.content }}
+      {% endfor %}
+    </div>
+  {% endif %}
```

Comment:
Just render the label without duplicating the rest please :)

Reply:
Fixed